string nodeName          #节点名称
string msgLog            #节点消息日志
bool Error               #错误·异常;   true : 有错误/异常。   false : 无错误/异常
string ErrorMsg          #错误/异常 详细消息